content/_data/changelogs/weekly.yml

```diff
@@ -264,7 +264,33 @@
       message: >
         Removed localizations with very low coverage: Albanian, Basque, Belarusian, Bengali, Esperanto, Galician, Georgian, Gujarati, Hindi, Icelandic, Indonesian, Irish, Kannada, Macedonian, Marathi, Mongolian, Occitan, Punjabi, Sinhala, Tamil, Telugu, Thai.
       pull: 2813
-
+- version: "2.53"
+  date: 2017-04-02
+  changes:
+    - type: major bug
+      issue: 42744
+      pull: 2826
+      message: >
+        Update to Windows Service Wrapper <tt>2.0.3</tt> and Windows Agent Installer <tt>1.8</tt> to prevent conversion of environment variables to lowercase in the agent executable, regression in Jenkins <tt>2.50</tt>.
+      references:
```

Comment:
FTR, that's how this should be done so it doesn't look weird:
https://github.com/jenkins-infra/jenkins.io/pull/779/files#diff-5313295831c35cd7b37d630b93438d60R256